tools/generate-classes/generate.js

```diff
@@ -576,7 +588,9 @@ function formatProperty(property) {
   result += `${property.type} ${property.cppSafeName}`;
 
   if (property.defaultValue !== undefined) {
-    result += " = " + property.defaultValue;
+    if (property.defaultValue != '""') {
```

Comment:
I think this would be better, though it doesn't change the generated code at all.
```suggestion
  if (property.defaultValue !== undefined && property.defaultValue != '""') {
```